Bio/EBI/ebisearch.py

```diff
@@ -0,0 +1,691 @@
+# This code is part of the Biopython distribution and governed by its
```

Comment:
Missing copyright statement with author names (and the dual licensing point raised separately).

Reply:
Sorry, this part is not clear. What years should I place for the copyright?
If I understood right it should be something like:
```
# Copyright 20xx-20xx Berenice Batut.  All rights reserved.
```

Reply:
See e.g. ``Bio/SeqRecord.py`` for our convention, the first line(s) are the copyright bits (Bernice as original author, and then revisions by you), followed by the license information.